composer.json

```
@@ -59,6 +59,9 @@
   "scripts": {
     "test": "phpunit"
   },
+  "provide": {
+    "ext-sodium": "*"
```

`ext-libsodium` can also be added. I first thought that composer 2 was not compatible with PHP < 7.0, but it is until its 2.2.x version.
```suggestion
    "ext-libsodium": "*",
    "ext-sodium": "*"
```